sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -428,14 +429,19 @@ def __init__(self, options):
       credentials = None
     else:
       credentials = get_service_credentials()
+
+    # Use 60 second timeout avoid hangs during network flakiness.
+    http_client = httplib2.Http(timeout=60)
```

Comment:
Could you update the comment to reflect what kind of timeout this is?
I believe it is a timeout at the socket operation level.

Reply:
Done. It is the socket timeout (https://github.com/httplib2/httplib2/blob/8b65b523359f0da7e91e130e4877f941cf4511d5/python2/httplib2/__init__.py#L1588)